### DISCUSSION TITLE
Question: How to create ExtractableDataSet in a YamlRepository using the CLI

### DISCUSSION THREAD

**Discussion by rkm** [HicServices/RDMP#1903] [Q&A] [answered]
I'm trying to bootstrap the SMI extraction pipeline using only a YamlRepository and a set of existing tables. So far, I've setup the logging server, imported all the TableInfos (including Catalogue creation), created JoinInfos between the Study/Series/Image tables for each modality, and set IsPrimaryExtractionTable.

When running the SMI CohortExtractor, no Catalogues are found. This is because it starts by looking for ExtractableDataSets. Looking at an export of a previous platform database, these appear to have a very simple format:

```yaml
Catalogue_ID: 1
DisableExtraction: false
Project_ID: 
ID: 1
```

I can create these manually, but is there any way to make them using the rdmp-cli?

**Comment by JFriel** [comment]
I don't believe there is currently a nice way to create them via the CLI. Will add a ticket and something like "rdmp.exe CreateExtractableDataset ..." should be available in the next release

**Comment by tznind** [comment, marked as answer]
Should be a command called like 'make catalog extractable' from memory

https://github.com/HicServices/RDMP/blob/develop/Rdmp.Core%2FCommandExecution%2FAtomicCommands%2FExecuteCommandChangeExtractability.cs

**Reply by rkm** [reply]
That worked - thanks Thomas!

**Comment by howff** [comment]
The source code link doesn't work any more, possibly moved to
https://github.com/HicServices/RDMP/blob/develop/Rdmp.Core/CommandExecution/AtomicCommands/ExecuteCommandMakeCatalogueItemExtractable.cs
